axelrod/strategies/inverse.py

```diff
@@ -7,7 +7,13 @@ class Inverse(Player):
     long ago the opponent defected."""
 
     name = 'Inverse'
-    memory_depth = float('inf')  # Long memory
+    behaviour = {
+        'memory_depth': float('inf'),  # Long memory
+        'stochastic': False,
```

Comment:
This one uses random, so seems like stochastic = True (though Player might catch it...)